spec/jekyll_seo_tag/drop_spec.rb

```diff
@@ -624,4 +624,23 @@
       end
     end
   end
+
+  context "canonical url" do
+    context "when canonical url is specified for a page" do
+      let(:page) { make_page( { "title" => "page title", "canonical_url" => "https://github.com/jekyll/jekyll-seo-tag/"} ) }
+      let(:canonical_url) { 'https://github.com/jekyll/jekyll-seo-tag/' }
+      it "uses specified canonical url" do
+        puts subject.canonical_url
```

Comment:
Forgot to remove puts, is this proper way to test or I need to create posts fixtures? I am not sure about this part :sweat_smile: